zproject/computed_settings.py

```diff
@@ -937,12 +949,12 @@ def zulip_path(path: str) -> str:
         },
         'zulip.zerver.webhooks': {
             'level': 'DEBUG',
-            'handlers': ['file', 'errors_file'],
+            'handlers': ['file', 'errors_file', 'webhook_file'],
             'propagate': False,
         },
         'zulip.zerver.webhooks.unsupported': {
             'level': 'DEBUG',
-            'handlers': ['file', 'errors_file'],
+            'handlers': ['file', 'errors_file', 'webhook_unsupported_file'],
```

Comment:
Do we want the additional locations on these?  I think the original intent at least had been to just do `webhook_unsupported_file`; we might have accidentally migrated that in some past logging refactor.  (The original point was to just record the giant block needed to investigate adding support out-of-band)

Reply:
Dropping them from the other logs makes sense; will do.